HotSpot/AppDelegate.m

```diff
@@ -20,8 +20,8 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
     
     ParseClientConfiguration *config = [ParseClientConfiguration   configurationWithBlock:^(id<ParseMutableClientConfiguration> configuration) {
         
-        configuration.applicationId = @"myAppId";
-        configuration.server = @"http://hotspot2017.herokuapp.com/parse";
+        configuration.applicationId = @"AaronInstagram17";
```

Comment:
you would need to change this before you merge with master (once Andrea has merged her data manager)